iris-mpc-cpu/src/hawkers/galois_store.rs

```diff
@@ -233,69 +233,91 @@ pub async fn gr_create_ready_made_hawk_searcher<R: RngCore + Clone + CryptoRng>(
     rng: &mut R,
     database_size: usize,
 ) -> eyre::Result<(
-    HawkSearcher<PlaintextStore, GraphMem<PlaintextStore>>,
-    HawkSearcher<LocalNetAby3NgStoreProtocol, GraphMem<LocalNetAby3NgStoreProtocol>>,
+    (PlaintextStore, GraphMem<PlaintextStore>),
```

Comment:
Would it be worth to create a struct for this tuple of tuples?

Reply:
not sure, since these are going to be used as individual arguments to the search algorithms :thinking:

Reply:
Fair enough, anyway it's just a nit